TusharKesarwani/Front-End-Projects
# feat: animated under construction page

# Description
This pull request addresses the implementation of an "Under Construction" page using HTML. It includes a title, basic styling, and an SVG graphic.

### Changes Made
- Added HTML structure for the "Under Construction" page.
- Included Bootstrap for basic styling.
- Integrated Google Fonts for typography.
- Incorporated Material Icons for the construction sign graphic.

### How to Test
1. Clone the repository.
2. Open the HTML file in a web browser.
3. Verify that the "Under Construction" page is displayed correctly.


## Type of change
- [✔ ] New feature (non-breaking change which adds functionality)
- [ X] This change requires a documentation update

# Checklist:
- [✔ ] I have taken help from some online resourses 
- [✔ ] My code follows the style guidelines of this project
- [✔ ] I have performed a self-review of my own code
- [ ✔] I have commented my code, particularly in hard-to-understand areas
- [ ✔] I have made corresponding changes to the documentation
- [✔ ] The title of my pull request is a short description of the requested changes.


# ATTACH SCREEN-SHOTS / DEPLOYMENT LINK
![screenshot-20240111-134811](https://github.com/TusharKesarwani/Front-End-Projects/assets/92971596/e707e92b-0cf0-4084-b1b0-313f3c54b49a)
